packages/expo-camera/ios/EXCamera/EXCamera.m

```diff
@@ -648,10 +648,6 @@ - (void)startSession
 #if TARGET_IPHONE_SIMULATOR
   return;
 #endif
-  if (![_permissionsManager hasGrantedPermissionUsingRequesterClass:[EXCameraPermissionRequester class]]) {
```

Comment:
do we want to remove this entirely? it seems it would be better to check that the appropriate permissions are available still and continue to invoke [`onMountingError`](https://docs.expo.io/versions/latest/sdk/camera/#onmounterror) -- but i don't have full context on this

Reply:
this check is performed on any call to `getPermissionsAsync()` but we don't know which ones the consuming app is going to require anymore, so making a check on mount no longer makes sense

Reply:
@ajsmth - we know for sure that if they're trying to mount a camera that they need camera permissions, but we could defer check for recording audio until video recording is attempted (which i think would be handled by the aforementioned `getPermissionsAsync()`)

Reply:
okay that's an easy change to make

Reply:
the reason it's useful to keep this here is that otherwise the app may crash without a meaningful error message